src/main/kotlin/com/bardsoftware/papeeria/backend/cosmas/CosmasGoogleCloudService.kt

```diff
@@ -142,4 +163,18 @@ class CosmasGoogleCloudService(private val bucketName: String,
             LOG.info("Deleting file failed: ${e.message}")
         }
     }
+
+    fun getPatchList(projectId: String, fileId: String): MutableList<CosmasProto.FileInformation.Patch>? {
+        return fileBuffer[projectId]?.get(fileId)?.patchesList
+    }
+
+    fun getPatchListFromMemory(fileId: String, version: Long): MutableList<CosmasProto.FileInformation.Patch>? {
```

Comment:
Alexandra, normally when programmer sees the words "from memory" he assumes that it is RAM. It is pretty surprising that a method which claims to get something from memory actually accesses persistent storage.

Reply:
ok, change name to ...FromStorage